spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala

```diff
@@ -33,6 +34,7 @@ object SpadeContext {
       config.getInt("max-attempt"),
       config.getInt("delivery-timeout"),
       config.getInt("execution-timeout"),
+      config.getString("log-uri"),
```

Comment:
is this required or optional?

Reply:
it's required

Reply:
can we make it optional? Orchard should have a default configuration there if missing.

Reply:
this is not for orchard logs, but job logs on s3. do we still wanna have a default in orchard?

Reply:
correct me if i'm wrong: data pipeline (orchard) doesn't know where job logs should be stored, it just takes command and execute it. hyperion (spade) decides where to put the logs as default. is that right?

Reply:
https://github.com/salesforce/orchard/blob/master/orchard-provider-aws/src/main/resources/reference.conf#L3 this is the job log URI that can be set as default (slightly different from AWS data pipeline), spade should be able to override it if it decides to do so.

Reply:
it's required by ssm.

Reply:
updated to optional